### ISSUE TITLE
Display Configured And Available Queues And Workers

### ISSUE THREAD

**Issue by simeonwillbanks** [stitchfix/resque-brain#16]
**Add view for configured and available queues and workers.**

These additions will provide an overview of the current configuration.

The queues overview can display registered queues with number of working jobs.

The workers overview can display running workers and their registered queues. 

Right now, it's only possible to see active processes (failed, running, and waiting jobs) and the schedule.

**Comment by simeonwillbanks** [comment]
[`ResqueInstance#running`](https://github.com/stitchfix/resque-brain/blob/master/app/models/resque_instance.rb#L27) is an example of worker introspection, and it may be useful for the *workers overview*.

**Comment by simeonwillbanks** [comment]
The "running" view may be a good place for a consolidated *workers and queues overview*.

![running](https://cloud.githubusercontent.com/assets/164506/7433199/cfcc9956-efe3-11e4-9502-0555bee14c98.png)

**Comment by simeonwillbanks** [comment]
Example *queues and workers overview* from [resque-web](https://github.com/resque/resque-web).

![e271i07](https://cloud.githubusercontent.com/assets/164506/7433391/9aff7bce-efe5-11e4-9a5d-979d7984298b.png)

![resque_workers](https://cloud.githubusercontent.com/assets/164506/7433394/a5f823a0-efe5-11e4-87e8-59f4a6472fe5.png)

**Comment by davetron5000** [comment]
Yeah, we could change "Running" to just "Workers" and either show 'em all like resque-web does, or sort the running ones at the top.
